squad-server/index.js

```diff
@@ -252,6 +252,48 @@ export default class SquadServer extends EventEmitter {
     await this.logParser.watch();
   }
 
+  async setupAdminList(remoteAdminLists) {
+    try {
+      const remoteAdmins = {
+        admins: {},
+        whitelist: {},
+        groups: {}
+      };
+
+      for (let idx = 0; idx < remoteAdminLists.length; idx++) {
+        const list = remoteAdminLists[idx];
+
+        const resp = await axios({
```

Comment:
I'd suggest moving this logic elsewhere on the basis that may want to support other methods of getting admin lists too, e.g. locally and perhaps over FTP, so this method might get quite long. Create a file in `utils` that exports a function called `fetchAdminLists` that takes the config, as shown below, that is then called within this method with the resulting being updated in the class' state. I'd also consider adding this like the `updatePlayerList`, etc. methods so it is updated over time (just less frequently or maybe even just after a map change).

Each admin list probably needs a type and details of the source, e.g.:
```
{ "type": "remote", "url": "HTTP://...." }
```
or 
```
{ "type": "local", "url": "path/to/file" }
```